### PR TITLE
ARQ-2237 Create a BOM for arquillian-jakarta

### PR DESCRIPTION
**Pull request by rhusar**
Resolves 
https://issues.redhat.com/browse/ARQ-2237

### CODE REVIEW

**Comment by rhusar** [comment]
@jamez please review.

**Review by jamezp — COMMENTED** [review]
Before I do a full review, is there a reason not use the the parent POM?

**Comment by rhusar** [comment]
> Before I do a full review, is there a reason not use the the parent POM?

I am not sure, but that's the way it is setup in arquillian-core's bom. I suppose that's intentional so that yet another pom isn't being brought in.

**Comment by jamezp** [comment]
> > Before I do a full review, is there a reason not use the the parent POM?
> 
> I am not sure, but that's the way it is setup in arquillian-core's bom. I suppose that's intentional so that yet another pom isn't being brought in.

Okay, no problem. It's not a huge deal, I was just curious :)

File: bom/pom.xml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    <version>1.8.1.Final-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    <name>Arquillian Jakarta BOM</name>
+    <url>https://arquillian.org</url>
+    <description>Arquillian Jakarta Bill Of Materials (BOM)</description>
+
+    <issueManagement>
+        <system>jira</system>
+        <url>https://issues.redhat.com/browse/ARQ</url>
+    </issueManagement>
```
Comment (jamezp) [inline bom/pom.xml]: IMO we should use:
```xml
<issueManagement>
    <system>GitHub</system>
    <url>https://github.com/arquillian/arquillian-jakarta/issues</url>
</issueManagement>
```

**Review by jamezp — CHANGES_REQUESTED** [review]

**Comment by rhusar** [comment]
> > > Before I do a full review, is there a reason not use the the parent POM?
> > 
> > 
> > I am not sure, but that's the way it is setup in arquillian-core's bom. I suppose that's intentional so that yet another pom isn't being brought in.
> 
> Okay, no problem. It's not a huge deal, I was just curious :)

I think the dependencyManagement section from the parent would leak into the bom, right? So no parents here.

File: bom/pom.xml (hunk 1 above)
Comment (rhusar) [reply bom/pom.xml]: Kind of ironic since I did create a Jira to track this issue :-) 

But we could just direct people to GH Issues if that's easier.

Comment (starksm64) [reply bom/pom.xml]: The issue tracker definitely needs to point to https://github.com/arquillian/arquillian-jakarta/issues as this is used by Jakarta EE projects outside of Red Hat

Comment (rhusar) [reply bom/pom.xml]: Will update.

Comment (rhusar) [reply bom/pom.xml]: Done, resolving discussion.

**Review by starksm64 — APPROVED** [review]

**Pull request merged**